src/jvm/test/org/jetbrains/kotlinx/lincheck/test/generator/ParamGeneratorsTests.kt

```diff
@@ -145,4 +145,28 @@ class StringParamGeneratorTest {
         assertTrue(generatesStringsLengths.groupBy { it }.size > 1)
     }
 
+}
+
+/**
+ * This test ensures that parameter generator dynamically expanding range
+ * will be shrunk to start size after each scenario run.
+ *
+ * [Corresponding issue description](https://github.com/Kotlin/kotlinx-lincheck/issues/179).
+ */
+class ParamGeneratorResetBetweenScenariosTest {
+
+    @Test
+    fun test() {
+        ModelCheckingOptions()
+            .threads(2)
+            .actorsPerThread(5)
+            .iterations(30)
+            .logLevel(LoggingLevel.INFO).check(this::class)
+    }
+
+    @Operation
+    @Suppress("UNUSED")
```

Comment:
Don't suppress methods in Lincheck tests. Add this suppress for the whole file instead.